framework/src/base/Assembly.C

```diff
@@ -174,13 +178,13 @@ Assembly::buildFaceFE(FEType type)
 }
 
 void
-Assembly::buildFaceNeighborFE(FEType type)
+Assembly::buildNeighborFE(FEType type)
 {
-  if (!_fe_shape_data_face_neighbor[type])
-    _fe_shape_data_face_neighbor[type] = new FEShapeData;
+  if (!_fe_shape_data_neighbor[type])
+    _fe_shape_data_neighbor[type] = new FEShapeData;
```

Comment:
@jiangwen84 - It looks like you didn't clean up this structure. I think that's your only problem.